mix.exs

```diff
@@ -35,6 +35,6 @@ defmodule ExFormat.Mixfile do
   #
   # Type "mix help deps" for more examples and options
   defp deps do
-    []
+    [{:ex_doc, "~> 0.14", only: :dev, runtime: false}]
```

Comment:
We're at `~> 0.16` now. Also, no need for `runtime: false`, it's fine if ex_doc makes it to the runtime in the dev environment.